libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -1399,12 +1400,12 @@ open class SalesforceSDKManager protected constructor(
      * Determines whether the device has a compact screen.
      * Taken directly from https://developer.android.com/guide/topics/large-screens/large-screen-cookbook#kotlin
      */
-    fun compactScreen(activity: Activity) : Boolean {
+    fun compactScreen(activity: Activity): Boolean {
```

Comment:
Also a rider to the bill, however the rest of this source is formatted to Android Studio defaults already.